### PR TITLE
None changeset

### PR DESCRIPTION
**Pull request by mykhailoserve**
Add none changeset option alongside major, minor, and patch prompts

Ignore none changeset in a CHANGELOG.md file `yarn changeset version`

### CODE REVIEW

File: packages/cli/src/commands/add/createChangeset.ts [hunk 1]
```diff
@@ -108,106 +164,40 @@ export default async function createChangeset(
...
+      const isLast = i === bumpTypes.length - 1;
+      let pkgsForThisBumpType: string[] = [];
 
-      let shouldReleaseFirstMajor = await confirmMajorRelease(pkgJson);
-      if (shouldReleaseFirstMajor) {
-        pkgsLeftToGetBumpTypeFor.delete(pkgName);
+      if (!packages.length) break;
 
-        releases.push({ name: pkgName, type: "major" });
+      if (isLast) {
```
Comment (pastelsky) [inline packages/cli/src/commands/add/createChangeset.ts:180]: Isn't very clear why things are different for the last bump type vs others to me. Could you explain?

**Review by pastelsky — COMMENTED** [review]
LGTM, except a minor clarification.

File: packages/cli/src/commands/add/createChangeset.ts [hunk 2]
```diff
@@ -108,106 +164,40 @@ export default async function createChangeset(
...
+        }
 
-      for (const pkgName of pkgsThatShouldBeMinorBumped) {
         pkgsLeftToGetBumpTypeFor.delete(pkgName);
-
-        releases.push({ name: pkgName, type: "minor" });
-      }
-    }
-
-    if (pkgsLeftToGetBumpTypeFor.size !== 0) {
```
Comment (mykhailoserve) [inline packages/cli/src/commands/add/createChangeset.ts:201]: @pastelsky 
> Isn't very clear why things are different for the last bump type vs others to me. Could you explain?

TL;DR:
For the last bump type, we don't need to call choosePkgsForBumpType

Details:
If you reached the last bump type (none) and there are still packages left - the changeset assumes that all these packages need to be bumped with the last bump (none) type, thus changeset doesn't ask a user to select packages.

**Review by pastelsky — APPROVED** [review]

**Comment by pastelsky** [comment]
CI has been enabled on this repo. Please update your PR against master / push a commit to trigger CI. Thanks!

**Comment by pastelsky** [comment]
@mykhailoserve are you able to sign the CLA and merge this?

**Comment by mykhailoserve** [comment]
@pastelsky I've signed, but CLA is still pending
Screenshot 2022-07-12 at 10 07 28

**Comment by pastelsky** [comment]
I think @VelikiyYaroslav  and @mpetrenko-atlas would have to sign it as well as they are contributors.

**Comment by VelikiyYaroslav** [comment]
> I think @VelikiyYaroslav and @mpetrenko-atlas would have to sign it as well as they are contributors.

Done.

**Comment by pastelsky** [comment]
@mykhailoserve all PRs are good to merge now.

**Comment by mpetrenko-atlas** [comment]
@pastelsky seems like I don't have write permissions anymore.

Screenshot 2022-07-15 at 10 32 43

**Comment by pastelsky** [comment]
@mpetrenko-atlas you should now.

**Comment by mpetrenko-atlas** [comment]
@pastelsky, still unable, could it be because I requested a review from @Blasz? (Can't merge any of my PRs)

Screenshot 2022-07-15 at 11 22 23
